src/data/data_download.py

```diff
@@ -75,10 +93,14 @@ def get_txt_data(myrepopath, panacearepopath, daily_list):
 
 
 def main_setup():
+    """
+    Main program that gathers data needed to scrape Tweet IDs from Twitter
+    via twarc.
+    """
     # set up path to current working directory & path to directory
     # containing Panacea data
-    homepath = setpath()
-    myrepopath = Path.cwd().parent.parent
+    homepath = Path.home()
```

Comment:
Will this always work, for any directory setup, or is this only intended to be used in exactly one location?